core/src/main/java/net/sourceforge/jnlp/util/WindowsDesktopEntry.java

```diff
@@ -151,9 +154,17 @@ private void manageShortcutList(ManageMode mode, String path) throws IOException
         if (!getWindowsShortcutsFile().exists()) {
             getWindowsShortcutsFile().createNewFile();
         }
-
+        LOG.debug("Using WindowsShortCutManager {}", getWindowsShortcutsFile().toString());
         if (ManageMode.A == mode) {
-            List<String> lines = Files.readAllLines(getWindowsShortcutsFile().toPath(), UTF_8);
+            List<String> lines = null;
+            // if UTF-8 fails, try ISO-8859-1
+            try {
+                LOG.debug("Reading Shortcuts with UTF-8");
+                lines = Files.readAllLines(getWindowsShortcutsFile().toPath(), UTF_8);
+            } catch (MalformedInputException me) {
+                LOG.debug("Reading Shortcuts with ISO-8859-1");
+                lines = Files.readAllLines(getWindowsShortcutsFile().toPath(), ISO_8859_1);
```

Comment:
ISO-8895-1 is a subset of UTF-8 thos if the file is encoded in ISO-8859-1 it can be paresd using UTF-8.
Therefote it makes no sense to fall back to ISO-8859-1.

Reply:
I have tested this the char that caused problem with UTF8 worked with ISO

Reply:
Appologies Janak. Only the first 128 characters of ISO-8859-1 are a subset of UTF-8.
Therefore yes some characters are different.
But if the file was written in CP-1252, wouldn't it make more sense to fall back to this encoding?

Reply:
Or maybe fall back to the default encoding of the machine. As the file was most likely previously written using the default encoding.

Reply:
Yes default makes more sense. On Windows default is windows-1252. Will make some tests and change.

Reply:
Changed and committed